Yoast/javascript
# Update replacement variable editor

## Summary
This PR makes some changes to the Replacement variable editor that allow us to use the editor in other places. Some of the styles in the editor have also been changed. This is done to reflect the new designs.

This will also change the style in the Snippet Editor once these changes will be merged. I discussed this with the team and we decided that it was better to keep a consistent style across the plugin.


This PR can be summarized in the following changelog entry:

* Adds default props to the Replacement Variable editor and change the styling of some of its components.

## TODO
Check the issue for work that still needs to be done.

## Test instructions

This PR can be tested by following these steps:
* This PR should not break any of the behaviour of the Snippet Editor. To test this do the following:
1. Install the premium plugin in a Wordpress Test environment.
1. Link the monorepo in the plugin to this branch.
1. Check out the snippet editor in the plugin. Does everything still work?
Think of:
	- Are the correct fields focussed when a title is clicked?
	- Does editing work as expected?
	- Does the style of the component fit in with the other components / styles?1

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * The Snippet Editor.

## UI changes

## Quality assurance


Fixes #487
